### ISSUE TITLE
Reorder tags

### ISSUE THREAD

**Issue by PoisonFrog** [Betterbird/thunderbird-patches#54]
One of the annoying things of Thunderbird is that you cannot reorder tags in the Preferences panel so you can't control which tag has the priority and is going to color the messages. Since messages can have multiple tags (useful for filtering/searching purposes) I think this feature would be a nice addition to Betterbird. There used to be extensions that did this but they are not compatible with newer versions. Afaik it was also possible to do this using about:config in older versions, but I don't think it is possible anymore.

**Comment by Betterbird** [comment]
That would be https://bugzilla.mozilla.org/show_bug.cgi?id=528034 from 2009.

**Comment by PoisonFrog** [comment]
> That would be https://bugzilla.mozilla.org/show_bug.cgi?id=528034 from 2009.

Exactly, I noticed sometimes tags reorder to alphabetical order but I don't know what triggers it. All I know is that it sucks to not be able to order them.

**Comment by Betterbird** [comment]
You should find some funding for all your requests. We already did the complex search terms (https://github.com/Betterbird/thunderbird-patches/issues/35) which was a lot of work. We're now focused on bringing out BB 102 in June. Besides, it's summer in Europa. So perhaps something for rainy days in the autumn.
